src/main/java/io/phdata/streamliner/schemadefiner/model/TableDiff.java

```diff
@@ -32,4 +37,33 @@ public TableDiff(
     this.existsInDestination = existsInDestination;
     this.existsInSource = existsInSource;
   }
+
+  public boolean allChangesAreCompatible(){
+    return columnDiffs.stream().allMatch(c -> c.getIsAdd());
+  }
+
+  public String columnDDL(
+      scala.collection.immutable.Map<String, scala.collection.immutable.Map<String, String>>
+          typeMapping) {
+    Map<String, Map<String, String>> javaTypeMap = JavaHelper.convertScalaMapToJavaMap(typeMapping);
+
+    List<String> list =
+        columnDiffs.stream()
+            .map(
+                columnDiff -> {
+                  String str = "";
+                  if (columnDiff.getIsAdd()) {
```

Comment:
If one change is not isAdd, then this will output two consecutive commas. I think we should do a filter before this map checking for isAdd.